di-ipv-cimit-stub/lambdas/stub-management/build.gradle

```diff
@@ -0,0 +1,41 @@
+plugins {
+	id "java"
+	id 'io.freefair.aspectj.post-compile-weaving' version '8.0.1'
+}
+
+group "uk.gov.di.ipv"
+
+repositories {
+	mavenCentral()
+}
+
+sourceCompatibility = 11
+targetCompatibility = 11
+
+dependencies {
+	implementation "com.amazonaws:aws-lambda-java-core:1.2.1",
+			"com.amazonaws:aws-lambda-java-events:3.11.1",
```

Comment:
are we using this lib ?

Reply:
Yes, are using in the StubManagementHandler for implements RequestHandler<APIGatewayV2HTTPEvent, APIGatewayV2HTTPResponse>